cmake-proxies/libcurl/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+include(ExternalProject)
+
+def_vars()
+
+set( CURL_DIR "${_INTDIR}/libcurl" )
+set( CURL_TAG "curl-7_76_0")
```

Comment:
Does this mean you will now create a release of audacity whenever [curl has another CVE](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=curl)? Have you considered the security implications that you open the door for in this scenario?

Reply:
Vendoring curl?! Of all dependencies? :scream: